docs/support/account.md

```diff
@@ -0,0 +1,73 @@
+# Account Management
+
+## Reset password - Vendor
+If your account is locked or forget your password, you can follow prompts to unblock yourself and get back to your account. 
```

Comment:
- Use "forgot"
- Use "follow the prompts"

Reply:
done